AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -9143,7 +9143,7 @@ depop.com#%#//scriptlet("set-cookie", "gdpr__google__analytics", "false")
 depop.com#%#//scriptlet("set-cookie", "gdpr__facebook__social", "false")
 kayak.de#%#AG_onLoad(function(){if(!window.location.href.includes('consent')){var b=new MutationObserver(function(){var a=document.querySelector('div[class$="-footer"] > span[class$="-decline"] > button');a&&a.click()});b.observe(document,{childList:!0,subtree:!0});setTimeout(function(){b.disconnect()},1E4)}});
 wz.de,volksfreund.de#%#AG_onLoad(function(){var a=new MutationObserver(function(){var b=document.querySelector('#consentAccept');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)});
-derstandard.at,derstandard.de#%#AG_onLoad(function(){var a=new MutationObserver(function(){var b=document.querySelector('button[title="Einverstanden"]');b&&(a.disconnect(),b.click())});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)});
+derstandard.at,derstandard.de#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('button[title="Einverstanden"]'); if(el) el.click(); }, 1500); });
```

Comment:
Better add the modified rule as a second one (as a backup) as sometimes site may take longer than 1,5 seconds to generate the code.